pyproject.toml

```
@@ -43,7 +43,7 @@ classifiers = [
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
 dependencies = [
-    "sqlalchemy ~= 1.4.0",
+    "sqlalchemy >= 1.4",
```

```suggestion
    "sqlalchemy >= 1.4.45",
```
Just for the sake of aligning this with ensembl-utils.